core/frontend/src/BingLocation.ts

```diff
@@ -27,6 +29,9 @@ export class BingLocationProvider {
     this._locationRequestTemplate = "https://dev.virtualearth.net/REST/v1/Locations?query={query}&key={BingMapsAPIKey}".replace("{BingMapsAPIKey}", bingKey);
     // this._localCircularSearchRequestTemplate = "https://dev.virtualearth.net/REST/v1/LocalSearch/?CircularMapView={{circle}}&key={BingMapsAPIKey}".replace("{BingMapsAPIKey}", bingKey);
   }
+  /** Return the location of a query (or undefined if not found)

```

Comment:
Can you elaborate on the expected contents of "query"?

Reply:
Yes, done.